src/components/commons/header/Header.js

```diff
@@ -22,6 +22,10 @@ export const HeaderNavLink = ({ to, children }: HeaderLinkProps) => (
     </h3>
 );
 
+export const HeaderWrapperNavLinks = ({ children }: React.Node) => (
```

Comment:
Oi @evelew ! Tá dando um erro na build do projeto por causa do parâmetro, creio eu que isso aqui possa solucionar:

```suggestion
export const HeaderWrapperNavLinks = ({ children: React.Node }) => (
```

Reply:
Ops, corrigido!

Reply:
Show, valeu @evelew